### PR TITLE
term : remove "Dvcfile" from versioning use case tutorial

### PR DESCRIPTION
**Pull request by sarthakforwet**
The following PR completes the below subtask from #1445. 
> versioning use case tutorial

### CODE REVIEW

File: content/docs/use-cases/versioning-data-and-model-files/tutorial.md [hunk 1]
```diff
@@ -317,26 +317,25 @@ When you have a script that takes some data as an input and produces other data
 > ```
 
 ```dvc
-$ dvc run -f Dvcfile \
+$ dvc run -n train \
           -d train.py -d data \
           -M metrics.csv \
           -o model.h5 -o bottleneck_features_train.npy -o bottleneck_features_validation.npy \
```
Comment (jorgeorpinel) [inline content/docs/use-cases/versioning-data-and-model-files/tutorial.md:323]: Noticed this line is too long. I'll fix it:
```suggestion
          -o model.h5 \
          -o bottleneck_features_train.npy \
          -o bottleneck_features_validation.npy \
```

File: content/docs/use-cases/versioning-data-and-model-files/tutorial.md [hunk 2]
```diff
@@ -317,26 +317,27 @@ When you have a script that takes some data as an input and produces other data
 > ```
 
 ```dvc
-$ dvc run -f Dvcfile \
+$ dvc run -n train \
           -d train.py -d data \
           -M metrics.csv \
```
Comment (jorgeorpinel) [inline content/docs/use-cases/versioning-data-and-model-files/tutorial.md:322]:
```suggestion
$ dvc run -n train -d train.py -d data \
```

File: content/docs/use-cases/versioning-data-and-model-files/tutorial.md [hunk 3]
```diff
@@ -317,26 +317,27 @@ When you have a script that takes some data as an input and produces other data
...
-          -o model.h5 -o bottleneck_features_train.npy -o bottleneck_features_validation.npy \
+          -o model.h5 \
+          -o bottleneck_features_train.npy \
+          -o bottleneck_features_validation.npy \
```
Comment (jorgeorpinel) [inline content/docs/use-cases/versioning-data-and-model-files/tutorial.md:325]:
```suggestion
          -o model.h5 -o bottleneck_features_train.npy \
          -o bottleneck_features_validation.npy -M metrics.csv \
```

File: content/docs/use-cases/versioning-data-and-model-files/tutorial.md [hunk 4]
```diff
@@ -317,26 +317,24 @@ When you have a script that takes some data as an input and produces other data
...
 
-Similar to `dvc add`, `dvc run` creates a
-[DVC-file](/doc/user-guide/dvc-files-and-directories) named `Dvcfile` (specified
-using the `-f` option). It tracks all outputs (`-o`) the same way as `dvc add`
-does. Unlike `dvc add`, `dvc run` also tracks dependencies (`-d`) and the
-command (`python train.py`) that was run to produce the result. We call such a
-DVC-file a "stage file".
+`dvc run` creates a pipeline stage named `train` (specified using the `-n`
+option) in [`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvcyaml-file)
+file. It tracks all outputs (`-o`) the same way as `dvc add` does. Unlike
```
Comment (jorgeorpinel) [inline content/docs/use-cases/versioning-data-and-model-files/tutorial.md:328]:
```suggestion
`dvc run` writes a pipeline stage named `train` (specified using the `-n`
option) in [`dvc.yaml`](/doc/user-guide/dvc-files-and-directories#dvcyaml-file).
It tracks all outputs (`-o`) the same way as `dvc add` does. Unlike
```

File: content/docs/use-cases/versioning-data-and-model-files/tutorial.md [hunk 5]
```diff
@@ -317,26 +317,24 @@ When you have a script that takes some data as an input and produces other data
...
+`dvc add`, `dvc run` also tracks dependencies (`-d`) and the command
+(`python train.py`) that was run to produce the result.
 
-> At this point you could run `git add .` and `git commit` to save the `Dvcfile`
-> stage file and its changed outputs to the repository.
+> At this point you could run `git add .` and `git commit` to save the updated
+> stage and its changed outputs to the repository.
```
Comment (jorgeorpinel) [inline content/docs/use-cases/versioning-data-and-model-files/tutorial.md:333]:
```suggestion
> At this point you could run `git add .` and `git commit` to save the `train`
> stage and its outputs to the repository.
```

File: content/docs/use-cases/versioning-data-and-model-files/tutorial.md [hunk 6]
```diff
@@ -317,26 +317,24 @@ When you have a script that takes some data as an input and produces other data
...
 
-`dvc repro` will run `Dvcfile` if any of its dependencies (`-d`) changed. For
-example, when we added new images to built the second version of our model, that
-was a dependency change. It also updates outputs and puts them into the
+`dvc repro` will run `train` stage if any of its dependencies (`-d`) changed.
```
Comment (jorgeorpinel) [inline content/docs/use-cases/versioning-data-and-model-files/tutorial.md:335]:
```suggestion
`dvc repro` will run the `train` stage if any of its dependencies (`-d`) changed.
```

Comment (jorgeorpinel) [reply content/docs/use-cases/versioning-data-and-model-files/tutorial.md:335]: Please watch basic grammar.

File: content/docs/use-cases/versioning-data-and-model-files/tutorial.md (hunk 4 above)
Comment (sarthakforwet) [reply content/docs/use-cases/versioning-data-and-model-files/tutorial.md:328]: run

Do we need to change it from here also?

Comment (jorgeorpinel) [reply content/docs/use-cases/versioning-data-and-model-files/tutorial.md:328]: Yes but it's out of scope. The whole ref. needs to be rewritten soon. I'm committing my suggestion above to save time. Please just fix the formatting. Resolving here.

Comment (jorgeorpinel) [reply content/docs/use-cases/versioning-data-and-model-files/tutorial.md]: Actually the command was already rewritten so yes, just this intro to that command should be rephrased... But still, out of scope for this PR. I'll send a separate one.

**Comment by jorgeorpinel** [comment]
Merged the restyled PR instead. Thanks @sarthakforwet

File: content/docs/use-cases/versioning-data-and-model-files/tutorial.md (hunk 4 above)
Comment (jorgeorpinel) [reply content/docs/use-cases/versioning-data-and-model-files/tutorial.md]: #1537

**Pull request merged**